android/app/build.gradle

```diff
@@ -68,6 +68,23 @@ dependencies {
     implementation 'io.reactivex.rxjava2:rxkotlin:2.4.0'
     implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
 
+
+    /**
+     * Base of data to visualisation class that gaves the ability
+     * to print plots (for example as a set of rectangles or lines)
+     */
+    implementation 'io.data2viz.d2v:scale:0.8.0-RC10'
+    implementation 'io.data2viz.d2v:core:0.8.0-RC10'
+    implementation 'io.data2viz.d2v:viz:0.8.0-RC10'
+    implementation 'io.data2viz.d2v:axis:0.8.0-RC10'
```

Comment:
I seems to me that we're not using data2viz in this PR.

Reply:
will remove